addon/components/polaris-card.js

```diff
@@ -62,6 +63,7 @@ export default Component.extend({
    * @property headerActions
    * @type {Action[]}
```

Comment:
This should probably get changed to `Object[]` now, maybe worth checking if any other `@type`s need updating too

Reply:
will leave for when we cleanup/refactor...just cause there are many things we'd need to update here